process/working_groups.md

```diff
@@ -0,0 +1,21 @@
+# ASWF Working Groups
+_Modeled off of CNCF Working Group model ( https://github.com/cncf/toc/blob/master/workinggroups/README.md )_
+
+## Introduction
+
+The purpose of working groups are to study and report on a particular question and make recommendations based on its findings. The end result of a working group may be a new project proposal, landscape, whitepaper or even a report detailing their findings. The intention of working groups is not to host a full project or specification. Working Groups can be formed at any time but must be sponsored by a [TAC member] and voted with a super majority vote by the ASWF TAC. The TAC can also shut down a working group with a super majority vote.
+
+## Process
+
+If you would like to submit a working group proposal, please submit the [template] to the TAC. Working groups can begin formation to complete the [template] before TAC approval, but will not have offical designation as an ASWF Working Group until TAC approval. You will also have to present to the ASWF TAC and wider community before your WG proposal will be voted upon by the TAC. 
+
+## FAQs
+
+Q: Who can propose a new working group?
+A: Anyone can propose a new working group ( whether from an ASWF member company or not ), but it must be sponsored by a [TAC member].
```

Comment:
I would suggest "member entity" rather than "member company" since some of the associate members like the Academy itself, VES, Blender or movielabs aren't companies per se.

Reply:
yes, or "member organization"